stripe-test-e2e/src/test/java/com/stripe/android/test/e2e/EndToEndTest.kt

```diff
@@ -11,16 +12,19 @@ import com.stripe.android.model.StripeIntent
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.runBlocking
 import org.junit.runner.RunWith
-import org.robolectric.RobolectricTestRunner
 import kotlin.test.Test
 
 @ExperimentalCoroutinesApi
-@RunWith(RobolectricTestRunner::class)
+@RunWith(AndroidJUnit4::class)
```

Comment:
I am curious if this actually runs faster or if it is still using Robolectric under the hood?

Reply:
According to https://developer.android.com/reference/androidx/test/runner/AndroidJUnit4

> Used when executed with standard junit runner. Will attempt to delegate to RobolectricTestRunner or delegate provided by android.junit.runner system property.